api/rpc/logs/logs.go

```diff
@@ -13,11 +15,16 @@ const (
 
 // Logs is used to implement log.LogServer
 type Logs struct {
-	ES elasticsearch.Elasticsearch
+	ES    elasticsearch.Elasticsearch
+	Store storage.Interface
```

Comment:
Services shouldn't know anything at all about where we store tokens, etc. Look at the example I put in #36 where there is a hypothetical auth package that handles these concerns.